alica_tests/src/test/test_alica_simple_plan_tree_serialization.cpp

```diff
@@ -0,0 +1,52 @@
+#include "test_alica.h"
+
+#include "Behaviour/Attack.h"
+#include "Behaviour/MidFieldStandard.h"
+#include <alica_tests/CounterClass.h>
+
+#include <alica/test/Util.h>
+#include <engine/AlicaClock.h>
+#include <engine/AlicaEngine.h>
+#include <engine/Assignment.h>
+#include <engine/BasicBehaviour.h>
+#include <engine/BehaviourPool.h>
+#include <engine/DefaultUtilityFunction.h>
+#include <engine/IAlicaCommunication.h>
+#include <engine/PlanBase.h>
+#include <engine/PlanRepository.h>
+#include <engine/TeamObserver.h>
+#include <engine/model/Behaviour.h>
+#include <engine/model/Plan.h>
+#include <engine/model/RuntimeCondition.h>
+#include <engine/model/State.h>
+
+#include <gtest/gtest.h>
+
+namespace alica
+{
+namespace
+{
+
+class AlicaSerialization : public AlicaTestFixture
+{
+protected:
+    const char* getRoleSetName() const override { return "Roleset"; }
+    const char* getMasterPlanName() const override { return "SimpleTestPlan"; }
+    bool stepEngine() const override { return false; }
+};
+
+/**
+ * Tests whether it is possible to run a behaviour in a primitive plan.
+ */
+TEST_F(AlicaSerialization, runBehaviourInSimplePlan)
+{
+    ASSERT_NO_SIGNAL
+    ae->start();
+    RunningPlan* rpRoot = ae->getPlanBase().getRootNode();
+    IdGrp msg;
+    _deepestNode = _rootNode;
+    _treeDepth = 0;
+    rpRoot->toMessage(msg, rpRoot, 0, 0);
+}
+} // namespace
+} // namespace alica
```

Comment:
missing newline 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline